Makefile

```diff
@@ -51,40 +51,11 @@ pylib:
 	cd $(py_dir); make python
 
 test:
-	remove_tmp_dir=0; \
-	if [ ! -d "$(example10_data_dir)" ]; then \
-	    mkdir $(example10_data_dir); \
-	    remove_tmp_dir=1; \
-	fi; \
-	cd $(examples_dir); \
-	timeout 20 python SRWLIB_Example10.py; \
-	code=$$?; \
-	RED='\033[0;31m'; \
-	GREEN='\033[0;32m'; \
-	NC='\033[0m'; \
-	if [ $$code -eq 0 ]; then \
-	    status='PASSED'; \
-	    color=$${GREEN}; \
-	    message=''; \
-	elif [ $$code -eq 124 ]; then \
-	    status='PASSED'; \
-	    color=$${GREEN}; \
-	    message=' (timeouted, expected)'; \
-	else \
-	    status='FAILED'; \
-	    color=$${RED}; \
-	    message=''; \
-	fi; \
-	echo -e "\n\tTest $${color}$${status}$${NC}. Code=$${code}$${message}\n"; \
-	rm -f $(example10_data_dir)/{ex10_res_int_se.dat,ex10_res_int_prop_se.dat,ex10_res_int_prop_me.dat}; \
-	if [ $$remove_tmp_dir -eq 1 ]; then \
-	    cd $(root_dir); \
-	    rm -rf $(example10_data_dir); \
-	fi;
+	timeout_s=120 bash $(examples_dir)/basic-test.sh
```

Comment:
This invokes the test script from @robnagler which should be more robust on different platform (get rid of an external dependency `timeout`, which does not exist on Mac).